shenyu-bootstrap/src/main/resources/application.yml

```diff
@@ -68,7 +68,8 @@ management:
 
 shenyu:
   matchCache:
-    enabled: false
+    selectorEnabled: false
+    ruleEnabled: true
```

Comment:
maybe add rule MaxFreeMemory ？

Reply:
maxFreeMemory means JVM can use available memory. if the memory summary is 512MB, and maxFreeMemory is 256MB, then the selector and rule cache can use 256MB.

Reply:
I meant to set up JVM memory separately

Reply:
selector give JVM selectorMaxFreeMemory, rule gives too? then the final maxFreeMemory = selectorMaxFreeMemory + ruleMaxFreeMemory.

Reply:
have 2 paramter， selector Memory and rule FreeMemory

Reply:
yes, I will change it